client/src/components/Tip/Tip.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import { Button, Popover, PopoverBody } from 'reactstrap';
+import PropTypes from 'prop-types';
+
+export const TIP_IMPORTANCE_LEVELS = {
+  NORMAL: 'normal',
+  HIGH: 'high',
+};
+
+const tipImportanceMap = {
+  [TIP_IMPORTANCE_LEVELS.NORMAL]: {
+    color: 'muted',
+    description: 'Tip',
+  },
+  [TIP_IMPORTANCE_LEVELS.HIGH]: {
+    color: 'danger',
+    description: 'Important tip',
+  },
+};
+
+class Tip extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+    };
+
+    this.handleTipToggle = this.handleTipToggle.bind(this);
+  }
+
+  handleTipToggle() {
+    this.setState((state) => ({ open: !state.open }));
+  }
+
+  render() {
+    const { content, fieldTitle, icon, id, importance } = this.props;
+    const { open } = this.state;
+
+    const tipIconColor = tipImportanceMap[importance].color;
+    const tipType = tipImportanceMap[importance].description;
+
+    return [
+      (
+        <Button
+          key={`${id}-tip-button`}
+          color="outline-secondary"
+          id={`${id}-tip`}
+          onClick={this.handleTipToggle}
+          className={`btn--no-text btn--last font-icon-${icon} text-${tipIconColor}`}
+          aria-label={`${tipType} for ${fieldTitle}`}
```

Comment:
I'll localise it, that was an oversight, but I disagree that it should be provided by the parent - this description is specifically tied into the importance of the tip and it can already be influenced via the `fieldTitle` prop.